port/unix/omrsysinfo.c

```diff
@@ -3369,6 +3425,29 @@ readCgroupFile(struct OMRPortLibrary *portLibrary, int pid, OMRCgroupEntry **cgr
 }
 
 /**
+ * Given a subsystem flag, returns enum for that subsystem
+ *
+ * @param[in] portLibrary pointer to OMRPortLibrary
+ * @param[in] subsystemFlag flag of type OMR_CGROUP_SUBSYSTEM_*
+ *
+ * @return value of type enum OMRCgroupSubsystem
+ */
+static OMRCgroupSubsystem
+getCgroupSubsystemFromFlag(uint64_t subsystemFlag)
+{
+	switch(subsystemFlag) {
+	case OMR_CGROUP_SUBSYSTEM_CPU:
+		return CPU;
+	case OMR_CGROUP_SUBSYSTEM_MEMORY:
+		return MEMORY;
+	default:
+		Trc_PRT_Assert_ShouldNeverHappen();
+	}
+	/* unreachable code */
+	Trc_PRT_Assert_ShouldNeverHappen();
```

Comment:
I think we need a return here and then just the assert in the default case should be sufficient,